smithy-aws-protocol-tests/model/restJson1/http-content-type.smithy

```diff
@@ -291,20 +291,41 @@ structure TestNoPayloadInputOutput {
 ///
 @readonly
 @http(uri: "/no_input_no_payload", method: "GET")
-operation TestNoInputNoPayload {
+operation TestGetNoInputNoPayload {
     output: TestNoPayloadInputOutput
 }
 
-apply TestNoInputNoPayload @httpRequestTests([
+@readonly
+@http(uri: "/no_input_no_payload", method: "POST")
+operation TestPostNoInputNoPayload {
```

Comment:
I'd say add TestPostNoPayload (similar to TestNoPayload but with POST) also. That would actually more closely represent the issue represented in the .NET issue.